### PR TITLE
doc/dev: refine "Concepts" 2 of 3

### PR DESCRIPTION
**Pull request by zdover23**
Refine the second third of the entries in the section "Concepts" in doc/dev/peering.rst.

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
- `jenkins test rook e2e`
</details>

### CODE REVIEW

File: doc/dev/peering.rst [hunk 1]
```diff
@@ -45,39 +45,36 @@ Concepts
...
+   a non-primary OSD in the *acting set* of a placement group. A replica has
+   been recognized as a non-primary OSD and has been *activated* by the
+   primary.
 
 *stray*
-   an OSD who is not a member of the current *acting set*, but
-   has not yet been told that it can delete its copies of a
-   particular placement group.
+   an OSD that is not a member of the current *acting set* and has not yet been
+   told that it can delete its copies of a particular placement group.
```
Comment (anthonyeleven) [inline doc/dev/peering.rst:54]: s/that it can/to/ ?

File: doc/dev/peering.rst [hunk 2]
```diff
@@ -45,39 +45,36 @@ Concepts
...
+   (or once existed) and also has a lower bound on *last epoch clean* or *last
+   epoch started*.
 
 *PG log*
-   a list of recent updates made to objects in a PG.
-   Note that these logs can be truncated after all OSDs
-   in the *acting set* have acknowledged up to a certain
-   point.
+   a list of recent updates made to objects in a PG. These logs can be
+   truncated after all OSDs in the *acting set* have been acknowledged.
```
Comment (anthonyeleven) [inline doc/dev/peering.rst:72]: have acknowledged the changes.

**Review by anthonyeleven — APPROVED** [review]

File: doc/dev/peering.rst (hunk 1 above)
Comment (zdover23) [reply doc/dev/peering.rst:54]: I like that better.

File: doc/dev/peering.rst (hunk 2 above)
Comment (zdover23) [reply doc/dev/peering.rst:72]: That's also better.

File: doc/dev/peering.rst (hunk 1 above)
Comment (anthonyeleven) [reply doc/dev/peering.rst:54]: "can' makes it sound like the OSD has a choice in the matter.

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/56724 - Squid backport
https://github.com/ceph/ceph/pull/56725 - Reef backport
https://github.com/ceph/ceph/pull/56726 - Quincy backport

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/56727 - refine "Concepts" 3 of 3

**Pull request merged**